src/store/donations/donation.actions.ts

```diff
@@ -11,6 +11,21 @@ export interface IFetchDonationsHistogramActionParams {
   token: string;
 }
 
+export interface IDeleteDonationActionParams {
+  id: number;
+  token: string;
+}
+
+export interface IUpdateDonationActionParams {
+  donationInput: Partial<IDonation>;
+  token: string;
+}
+
+interface IFetchDonationsResult {
```

Comment:
This one feels like it belongs to donations? With an s, plural.

Reply:
It's still there, I just copied it because it was used as the result of the deletion action. But it's better to just import it, thanks.